lib/queryBuilder/graphInserter/DependencyGraph.js

```diff
@@ -27,10 +27,10 @@ class DependencyGraph {
 
     if (Array.isArray(models)) {
       for (let i = 0, l = models.length; i < l; ++i) {
-        this.buildForModel(modelClass, models[i], null, null, this.allowedRelations);
+        this.buildForModel(modelClass, models[i], null, null, this.allowedRelations, '');
```

Comment:
Empty string should be an empty array instead right?

Reply:
Yes! That's an oversight, with no consequences, because [...emptyString] = [] : )